web/src/utils/getSnapPoints.ts

```diff
@@ -8,7 +8,7 @@ const mapIconsHeight = 60
 const getLargeSnapPoint = (maxHeight: number) =>
   maxHeight - dimensions.headerHeightLarge - dimensions.navigationMenuHeight - mapIconsHeight
 
-export const getSnapPoints = (maxHeight: number): number[] => [
+export const getSnapPoints = (maxHeight: number): [number, number, number, number] => [
```

Comment:
Could we now remove the non-null assertions for getSnapPoins?

Reply:
sadly not all of them, as `react-map-gl` uses `number[]` as a type.